### PR TITLE
Clarify possible sources of action data in a P4 program.

### PR DESCRIPTION
**Pull request by jnfoster**

### CODE REVIEW

**Comment by mbudiu-vmw** [comment]
Fixes #914

File: p4-16/spec/P4-16-spec.mdk [hunk 1]
```diff
@@ -6240,7 +6242,7 @@ table smac {
 ~ End P4Example
 
 - The entries in the `smac` `table` may contain two different actions: `Drop_action` and `Rewrite_mac`.
-- The `Rewrite_smac` action has one parameter, `sourceMac`, which is bound by the control plane.
+- The `Rewrite_smac` action has one parameter, `sourceMac`, which will be provided in the table entry.
```
Comment (mbudiu-vmw) [inline p4-16/spec/P4-16-spec.mdk:6245]: Is this change in the wrong place? For this example the data is bound by the control-plane.

Comment (jnfoster) [reply p4-16/spec/P4-16-spec.mdk]: Fixed. Thanks.

**Review by mbudiu-vmw — APPROVED** [review]

**Comment by mbudiu-vmw** [comment]
We have agreed at the LDWG to merge this.

**Review by jafingerhut — COMMENTED** [review]
Looks good to me.

**Pull request merged**